### DISCUSSION TITLE
Videos not popping up when running workflow

### DISCUSSION THREAD

**Discussion by MahnoorK1** [bonsai-rx/bonsai#1020] [Q&A] [answered]
Hello, 

I am a first-time user and have a few questions with regard to my current workflow (image attached) with two cameras. Any help is very appreciated!

1. My two cameras are connected to the laptop and are associated with the two different videocapturedevice nodes. Once I start the workflow, although it says it's 'running', there's no indication for it - no pop-up of the videos being captured from the two cameras. When I stop the workflow, I can see the two different avi. files in my laptop folder. Is there a reason why the pop-ups are not showing up? It would help to adjust my cameras if I could have a preview of how it was looking as it was recording. It used to show the pop-up when I first started but stopped doing it all of a sudden.

2. What's the difference between cameracapture and videocapturedevice node? I've seen a few videos where camera capture is used to get video rather than video capture. Should I be using one over the other? 

Thank you for any help!

MK
![Bonsai screenshot](https://user-images.githubusercontent.com/112672417/188042428-b455908c-2c53-483e-9489-26721ec90f9a.jpeg)

**Comment by bruno-f-cruz** [comment, marked as answer]
Heya.

Make sure you have the `'Starter Package'` installed, and double-click one of the nodes while Bonsai is running. This should open a visualizer window.

Regarding `CameraCapture` vs `VideoCaptureDevice`. If I am not mistaken, the former implements the OpenCV's VideoCapture functionality, while the latter uses generic DirectX's drivers to access the camera.

Best,
Bruno

**Reply by MahnoorK1** [reply]
Thank you for your help, will try that out!
